Workflows/Environment Synchronization.md

```diff
@@ -0,0 +1,126 @@
+## Environment Synchronization
+A synchronization of environments, or more precisely, a synchronization of the data used in different environments,
+is usually necessary when you need to work in an environment that is as production-like as possible, but you don't have
+access to production itself, for example:
+* when you need to debug a more complex issue occuring in production
+* when you need data that is as close to production as possible (both when it comes to type and amount)
+for development and/or testing purposes
+
+Access to production is normally limited in order to protect the private information of users, among other things.
+The same should be ensured once this data is copied to another environment. In order to limit the number
+of developers who have access to raw production data, it would be best if that data was anonymized by someone
+from the DevOps team before it is given to the developer. To achieve this, the Rails team has agreed with
+the DevOps team that the synchronization process should look like this:
+1. a completely new environment is set up
```

Comment:
When we say completely new, can it be an existing staging/uat one?

Reply:
Technically it could be staging or UAT, but the idea is that the production database in its raw state is copied to an env that the developer doesn't have access to and that it is anonymized there before being put in any place that can be accessed by someone other than a devops person.

Reply:
Ahh so its not a fully fledged environment, just enough to copy over the DB, do work on it and then push to to S3 👍